public/index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8"/>
+  <meta name="viewport" content="width=device-width, initial-scale=1"/>
+  <title>F1ormula</title>
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bulma/0.6.0/css/bulma.min.css"/>
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
+  <link rel="stylesheet" href="styles.css"/>
+</head>
+<body>
+
+  <section class="hero">
+    <div class="hero-body">
+      <div class="container brand">
+        <img id="f1ormula-brand" src="images/brand/f1ormula_brand_no-flag-01.png" alt="F1ormula" />
+      </div>
+    </div>
+  </section>
+  <section>
+    <div class="field">
```

Comment:
Does this drop down do anything yet? If not, you should remove it until it is functional. Perhaps in another feature.